packages/performance-tests/test/TestContext.ts

```diff
@@ -0,0 +1,84 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+
+import { BriefcaseDb, BriefcaseManager, IModelHost, RequestNewBriefcaseArg } from "@itwin/core-backend";
+import { Logger } from "@itwin/core-bentley";
+import { BriefcaseIdValue } from "@itwin/core-common";
+import { AccessTokenAdapter, BackendIModelsAccess } from "@itwin/imodels-access-backend";
+import assert from "assert";
+
+const loggerCategory = "TestContext";
+
+export interface TestIModel {
+  name: string;
+  iModelId: string;
+  iTwinId: string;
+  tShirtSize: string;
+  load: () => Promise<BriefcaseDb>;
+}
+
+export const testITwinIds = [
+  "892aa2c9-5be8-4865-9f37-7d4c7e75ebbf",
```

Comment:
let's move this to some sort of configuration file and not include literal ids here